generators/app/index.js

```diff
@@ -4,8 +4,9 @@ var path = require('path');
 var yeoman = require('yeoman-generator');
 var yosay = require('yosay');
 var chalk = require('chalk');
-var npmName = require('npm-name');
+var npm = require('npm');
```

Comment:
What's the motivation for changing to npm instead of npmName? Mostly curious. Also, would be able to remove it as a dependency if it's not being used anymore.